backend/src/install/cleanup.rs

```diff
@@ -342,6 +342,11 @@ where
     for<'a> &'a mut Ex: Executor<'a, Database = Postgres>,
 {
     let mut tx = db.begin().await?;
+    crate::db::DatabaseModel::new()
+        .package_data()
+        .idx_model(&id)
+        .lock(&mut tx, LockType::Write)
```

Comment:
Lower down we get the package as a mut. Again I believe that this system used to lock the package. Now we are making this more explicit.